examples/stocks/lib/main.dart

```diff
@@ -94,6 +94,13 @@ class StocksAppState extends State<StocksApp> {
     return null;
   }
 
+  Future<LocaleQueryData> _onLocaleChanged(ui.Locale locale) {
+    return initializeMessages(locale.toString()).then((_) {
+      Intl.defaultLocale = locale.toString();
+      return StockStrings.instance;
```

Comment:
StockStrings has no state - it's just a collection of static Intl.message calls.  So I allocate a single instance and return it here so it can be passed onward to widgets.

Is there a better alternative?